pkgs/applications/blockchains/alfis/default.nix

```diff
@@ -27,10 +26,16 @@ rustPlatform.buildRustPackage rec {
     "--skip=dns::client::tests::test_udp_client"
   ];
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config makeWrapper ];
   buildInputs = lib.optional (withGui && stdenv.isLinux) webkitgtk
     ++ lib.optionals (withGui && stdenv.isDarwin) [ Cocoa WebKit ];
 
+  postInstall = lib.optionalString (withGui && stdenv.isLinux) ''
+    cp $out/bin/alfis{,_unwrapped}
+    makeWrapper $out/bin/alfis{_unwrapped,} \
+      --prefix PATH : ${lib.makeBinPath [ dialog ]}
```

Comment:
```suggestion
    makeWrapper $out/bin/alfis \
      --prefix PATH : ${lib.makeBinPath [ dialog ]}
```

You are sure that you have this diff?

Reply:
Oh, I see. We need to change `makeWrapper` to `wrapProgram`. Didn't know about that.